cli/Cargo.toml

```diff
@@ -43,7 +43,7 @@ sentry = { version = "0.32.3", features = ["test"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1"
 serde_with = "3.8.2"
-serde_yaml = "0.9"
+serde_yml = "0.0.10"
```

Comment:
blocking: per @ysndr 's comment, as the author of `serde_yaml` is a lot more trustworthy than that of `serde_yml` (at least to my knowledge) and `serde_yml` has only been maintained 4 days longer, I don't think we should change this yet

Reply:
Done in b223d5d.